Whisky/Models/BottleVMEntries.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  BottleVMEntries.swift
+//  Whisky
+//
+//  Created by Josh on 6/22/23.
+//
+
+import Foundation
+
+struct BottleEntries: Codable {
+    var fileVersion: Semver = Semver(major: 1, minor: 0, patch: 0)
+    var paths: [URL] = []
+}
+
+class BottleVMEntries {
+    static let containerDir = FileManager.default.homeDirectoryForCurrentUser
+        .appendingPathComponent("Library")
+        .appendingPathComponent("Containers")
+        .appendingPathComponent(Bundle.main.bundleIdentifier ?? "com.isaacmarovitz.Whisky")
+    static let bottleEntriesDir = containerDir
+        .appendingPathComponent("BottleVM")
+        .appendingPathExtension("plist")
+    private var file: BottleEntries {
+        didSet {
+            encode()
+        }
+    }
+    var paths: [URL] {
+        get {
+            file.paths
+        }
+        set {
+            file.paths = newValue
+        }
+    }
+    static func exists() -> Bool {
+        return FileManager.default.fileExists(atPath: Self.bottleEntriesDir.path())
+    }
+    init() {
+        file = .init()
+        if !Self.exists() {
```

Comment:
The reason this is needed is beacuse the bottleVM creates this instance and it will only upgrade the files if it detects no folder.